pkg/storage/s3/s3.go

```diff
@@ -129,86 +253,68 @@ func (d *driver) createAndTagBucket(svc *s3.S3, installConfig *installer.Install
 	encryptionConfig := &s3.ServerSideEncryptionConfiguration{Rules: encryptionRules}
 	bucketEncryptionInput := &s3.PutBucketEncryptionInput{Bucket: aws.String(d.Config.Bucket), ServerSideEncryptionConfiguration: encryptionConfig}
 
-	_, err := svc.PutBucketEncryption(bucketEncryptionInput)
+	_, err = svc.PutBucketEncryption(bucketEncryptionInput)
 	if err != nil {
-		return err
+		if aerr, ok := err.(awserr.Error); ok {
+			// If we can't set default encryption on the bucket don't return an error
+			// just set the StorageEncrypted condition to false
```

Comment:
what can the user/admin do to get encryption added to the bucket later?  Do/can they just go encrypt the bucket manually?  Does anything else need to be updated if they do that (e.g. on the registry configuration?).

Is/should there be a way for them to force the operator to attempt to encrypt the bucket again once they've resolved the issue?  (my inclination is to say yes, eventually, but probably not for 4.0).

Reply:
The user could log into the aws console and turn on default encryption if they want to, and nothing in the registry config needs to be changed.

We could add an option to the config that says the storage should be encrypted, and we check to see if it's encrypted on sync and if it's not, we attempt to encrypt it, if the condition is false and shouldEncrypt is true or something.